### ISSUE TITLE
Info command enhancements needed

### ISSUE THREAD

**Issue by Bilal-S** [serverless/serverless#1926]
##### Serverless Framework Version:

1.0
##### Operating System:

Any
##### Expected Behavior:

The info command allows discovery of local status of service as well as remote status. A flag could indicate if only one part or the other is needed.
`serverless info --[all|remote|local]`
The local info should indicate all configuration for the service, the function names, the endpoint configuration (the request/response template files). A flag indicating deployed vs non-deployed for each function would be helpfull as well.

The remote config should have the current info, i.e. what remote functions are deployed and their endpoints.
##### Actual Behavior:

The info command provides status of local as well as remote configuration. Currently the info command only focus is remote endpoints. This command could be an important tool similar to `git status` in determining the life cycle of code deployment. As is, its usefulness is very limited.
##### Feature Request:

please see expected behavior
##### Benefits:
- Allow the status of lifecylce  of deployment
- Get idea of which functions are in progress and new 
- in teams a call to info would tell you the difference between local and remote

Discussion template  ----------------------------------------------
##### TLDR:
- an improved `serverless info` command can help with development and become a useful tool in any project similar to `git status`

**Comment by flomotlik** [comment]
At the moment we only want info to show what is in the deployed environment as diffing and such is a lot more complex and will be included in the deploy tooling probably so I'm going to close.
